Tasks/UsePythonVersion/task.json

```diff
@@ -0,0 +1,57 @@
+{
+    "id": "33C63B11-352B-45A2-BA1B-54CB568A29CA",
+    "name": "UsePythonVersion",
+    "friendlyName": "Use Python Version",
+    "description": "Retrieves the specified version of Python from the tool cache. Optionally add it to PATH.",
+    "helpMarkDown": "",
+    "category": "Tool",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 132,
+        "Patch": 0
+    },
+    "demands": [],
+    "instanceNameFormat": "Use Python $(versionSpec)",
+    "inputs": [
+        {
+            "name": "versionSpec",
+            "type": "string",
+            "label": "Version spec",
+            "required": true,
+            "defaultValue": ">= 3.3",
+            "helpMarkDown": "Version range or exact version of a Python version you want to use."
+        },
+        {
+            "name": "outputVariable",
+            "type": "string",
+            "label": "Output variable",
+            "required": true,
+            "defaultValue": "",
```

Comment:
We should set a default value for the outputVariable

Reply:
I wanted to do something like PYTHON_$(versionSpec), but it seems like we don't have the capability to do that kind of processing in the UI.  I'll keep thinking about it.